tests/test_lammps_spin.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from context import dpdata
+
+
+class TestLmp(unittest.TestCase):
+    def test_dump_input(self):
+        tmp_system = dpdata.System(
+            os.path.join("poscars", "conf.lmp"), type_map=["O", "H"]
+        )
+        tmp_system.data["spins"] = [[[3, 4, 0], [0, 4, 3]], []]
```

Comment:
why it is a list of length 2 and the 2nd element is `[]`?

Reply:
Sorry, this is a mistake